packages/create/templates/demo/package.json

```diff
@@ -14,7 +14,7 @@
 		"@sveltejs/adapter-auto": "^4.0.0",
 		"@sveltejs/kit": "^2.9.0",
 		"@sveltejs/vite-plugin-svelte": "^5.0.0",
-		"svelte": "^5.0.0",
+		"svelte": "^5.25.0",
```

Comment:
I think we need to update https://github.com/sveltejs/cli/blob/b53be6269bbac784bfd60709c5274c1cf4edc461/packages/create/templates/demo/package.template.json#L18